### PR TITLE
Avg,std dev

### PR DESCRIPTION
**Pull request by Fawada**
So the count function is renamed and claculates various operations depending upon the value of argument "calc". So is this what you are looking for?

### CODE REVIEW

File: meerkat_analysis/indicators.py [hunk 1]
```diff
@@ -41,13 +41,23 @@ def fix_dates(start_date, end_date, epi_week_start_day):
     return start_date, end_date, freq
 
 
-def count(data, var_id, start_date=None, end_date=None, epi_week_start_day=None):
+def calculate(data, var_id,calc,start_date=None, end_date=None, epi_week_start_day=None):
```
Comment (jonathansberry) [inline meerkat_analysis/indicators.py:44]: PEP8: just missing a space after `var_id,` and after `calc,`

File: meerkat_analysis/indicators.py [hunk 2]
```diff
@@ -41,13 +41,23 @@ def fix_dates(start_date, end_date, epi_week_start_day):
...
     """
     We return the total count of var_id and a timeline by epi_week
 
     Args:
         data: data in dataframe
         var_id: the variable id to count
+        calc: the mathematical function to calculate
+        ot: overall total
```
Comment (jonathansberry) [inline meerkat_analysis/indicators.py]: I think you want to indent all the possible values of the variable `calc`? Just so it is clear that they are possible values of "calc" and not extra arguments of the function calculate.

File: meerkat_analysis/indicators.py [hunk 3]
```diff
@@ -60,20 +70,32 @@ def count(data, var_id, start_date=None, end_date=None, epi_week_start_day=None)
                                            epi_week_start_day)
 
     dates = pd.date_range(start_date, end_date, freq=freq, closed="left")
-
+    return_val=0.0
+    if calc==ot:
```
Comment (jonathansberry) [inline meerkat_analysis/indicators.py]: `if calc==ot:` Does this work? You don't seem to have defined the `ot` variable anywhere. 
Shouldn't it be `if calc is "ot":`?

In fact if you were super keen you could use a dictionary based model instead. Would be quite a bit neater I think:
```
calculation = {
    "ot": pd[var_id].sum(),
    "om": pd[var_id].mean(),
    "omin": pd[var_id].min()
}
[...]
return calcuation[calc], timeline
```

**Review by jonathansberry — CHANGES_REQUESTED** [review]
This is solving the problem as I understand it. However, I'm not sure this code currently works? Also, there might be a neater way of doing the switch using a dictionary, instead of many if statements. I'll defer to Gunnar to check that we have the right understanding of what the function should do.

File: meerkat_analysis/indicators.py (hunk 1 above)
Comment (Fawada) [reply meerkat_analysis/indicators.py:44]: Yes, it was the starting point  to understand the problem, I will make it neat and add switch statement etc.

**Pull request closed without merging**